packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -157,7 +157,7 @@ module.exports = async (api, args = {}, pluginSource) =>
     })
 
     // Check that the API is documented.
-    if (!apiList[api]) {
+    if (!apiList[api] && api !== `FAKE_API_CALL`) {
```

Comment:
I can't see `FAKE_API_CALL` if you could point me in the right direction that would be awesome

Reply:
Oh lemme add a comment on this. It's a bit weird :-D

Reply:
I missed https://github.com/gatsbyjs/gatsby/pull/11831/files#diff-deed9a141e906a882e5f3b2d2c4e21aeR19 my bad :) it makes sense